<!-- xml -->
<launch>

  <!-- Run the map server -->
  <arg name="map_file" default="/datos/bamboo_ros_ws/maps/oficina.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" >
    <param name="frame_id" value="/map" /> 
  </node>

  <group ns="robot1">
    <arg name="_robot_name"  value="robot1" />
    <arg name="_tf_robot" value="$(arg _robot_name)_tf" /> 
    <param name="tf_prefix" type="string" value="$(arg _tf_robot)"/>
    
    <include file="$(find multi_robot_collabration)/launch/move_base.launch">
     <arg name="robot_name" value="$(arg _robot_name)"/>
     <arg name="tf_robot" value="$(arg _tf_robot)" /> 
     
    </include>
    
    <node name="robot_pose_publisher" pkg="robot_pose_publisher" type="robot_pose_publisher" output="screen">      
      <param name="base_frame"             value="$(arg _tf_robot)/base_link"/> 
    </node> 
 
  </group>
  
  <group ns="robot2">
    <arg name="_robot_name"  value="robot2" />
    <arg name="_tf_robot" value="$(arg _robot_name)_tf" /> 
    <param name="tf_prefix" type="string" value="$(arg _tf_robot)"/>
    
    <include file="$(find multi_robot_collabration)/launch/move_base.launch">
     <arg name="robot_name" value="$(arg _robot_name)"/>
     <arg name="tf_robot" value="$(arg _tf_robot)" /> 
     
    </include>
     <node name="robot_pose_publisher" pkg="robot_pose_publisher" type="robot_pose_publisher" output="screen">      
      <param name="base_frame"             value="$(arg _tf_robot)/base_link"/> 
    </node> 
    
  </group>
<group ns="robot3">
    <arg name="_robot_name"  value="robot1" />
    <arg name="_tf_robot" value="$(arg _robot_name)_tf" /> 
    <param name="tf_prefix" type="string" value="$(arg _tf_robot)"/>
    
    <include file="$(find multi_robot_collabration)/launch/move_base.launch">
     <arg name="robot_name" value="$(arg _robot_name)"/>
     <arg name="tf_robot" value="$(arg _tf_robot)" /> 
     
    </include>
    
    <node name="robot_pose_publisher" pkg="robot_pose_publisher" type="robot_pose_publisher" output="screen">      
      <param name="base_frame"             value="$(arg _tf_robot)/base_link"/> 
    </node> 
 
  </group>
  
</launch>
